source/SpinalHDL/Data types/bits.rst

```diff
@@ -282,11 +307,15 @@ Misc
      - Return the index of the MSB (highest allowed index for Bits)
      - Int
    * - x.msb
-     - Return the most significant bit
+     - Return the most significant bit value
      - Bool
    * - x.lsb
-     - Return the least significant bit
+     - Return the least significant bit value
      - Bool
+   * - x.reverse
+     - Return a representation of the same instance in reverse bit order,
```

Comment:
```suggestion
     - Return a copy of x with reversed bit order
```
Use the same wording as below?

Reply:
I see what you mean about the use of the word `copy` in the item below or so.  I would argue this is also a bad choice and the word `representation` is a better word there as well, relating to a `data representation` of the same underlying thing.  In reading the docs I was trying not to be too nit-picky over such things.  But if I were I would change that too and use a distinct language word less over-used than the word copy (can mean a lot of different things, so lots of room for misunderstanding).

Reply:
I had a quick look at the code: it looks to me like this should actually be `reversed` - overlooked that initially
https://github.com/SpinalHDL/SpinalHDL/blob/af55caaf3e9821eed2b2ad2d5bfda742716bcb43/core/src/main/scala/spinal/core/Bits.scala#L237
I used copy since the reversed vector will actually not let you modify the original one, i.e.
```scala
val x = Bits(5 bit)
val y = x.reversed
when(x(0)) { y(3) := True }
```
generates
```verilog
y = {x[0],{x[1],{x[2],{x[3],{x[4]}}}}}
if(...) begin
  y[3] = 1'b1;
end
```